src/difference/difference_solution1/README.md

```diff
@@ -0,0 +1,83 @@
+<!-- BEGIN DOCS -->
+# Difference: finds the numbers that are in the first array but not in the second array
```

Comment:
Which numbers is going to find? I see more like a function that compares the values of one array with another and returns the value that differs from the first array with the second array

Reply:
Read me file description for the function is as follows: "Creates an array of values that are in the first array, but not not in the second array."

and the given examples are : "@example
 *
 * difference([2, 1], [2, 3]); //  [1]
 *
 * @example
 *
 * difference([1, 2, 1], [2, 3]); //  [1]"  so as the main rule of the TDD write the least code that passes tests. Thus only number are included and the definition is clearified.